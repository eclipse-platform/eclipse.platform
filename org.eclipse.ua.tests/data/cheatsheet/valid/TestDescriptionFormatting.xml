<?xml version="1.0" encoding="UTF-8" ?> 
<!--
     Copyright (c) 2005, 2006 IBM Corporation and others.

     This program and the accompanying materials
     are made available under the terms of the Eclipse Public License 2.0
     which accompanies this distribution, and is available at
     https://www.eclipse.org/legal/epl-2.0/

     SPDX-License-Identifier: EPL-2.0
    
     Contributors:
         IBM Corporation - initial API and implementation
 -->

<cheatsheet title="Testing Infopop and Help Links">

<intro 
	href="/org.eclipse.platform.doc.user/reference/ref-cheatsheets.htm">
	<description>
This cheat sheet will test infopop and help link.
Let's get started!
	</description>
</intro>

<item title="Item with simple text">
<description>
Some simple text to test the decription formating.
</description>
</item>

<item title="Item with a line break"><description>
Line one
<br/>
Line two with a special &lt; char in the middle.
</description></item>

<item title="Item with all the special charaters"><description>
&lt; - less than
<br/>
<b>Special characters</b><br/>
less than:   &lt;
<br/>
greater than: &gt;
<br/>
ampersand:   &amp;
<br/>
apostrophe:  &apos;
<br/>
quote:       &quot;
<br/>
</description></item>

<item title="Item with one less than"><description><b>&lt;</b></description></item>
<item title="Item with one greater than"><description>&gt;</description></item>
<item title="Item with one bold less than"><description><b>&lt;</b></description></item>
<item title="Item with one bold greater than"><description><b>&gt;</b></description></item>
<item title="Item with one less than before a break"><description>&lt;<br/></description></item>
<item title="Item with one less than after a break"><description><br/>&lt;</description></item>
<item title="Item with one less than between breaks"><description><br/>&lt;<br/></description></item>
<item title="Item with one greater than before a break"><description>&gt;<br/></description></item>
<item title="Item with one greater than after a break"><description><br/>&gt;</description></item>
<item title="Item with one greater than between breaks"><description><br/>&gt;<br/></description></item>


</cheatsheet>
