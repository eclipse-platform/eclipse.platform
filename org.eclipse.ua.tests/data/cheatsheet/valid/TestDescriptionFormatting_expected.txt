<?xml version="1.0" encoding="UTF-8" ?>
<cheatsheet
      title="Testing Infopop and Help Links">
   <item
         title="Introduction"
         description="This cheat sheet will test infopop and help link.
Let's get started!"
         Href="/org.eclipse.platform.doc.user/reference/ref-cheatsheets.htm"
         contextId="null">
      <nullAction/>
      <nullList/>
      <nullPerformWhen/>
      <nullList/>
   </item>
   <list>
      <item
            title="Item with simple text"
            description="Some simple text to test the decription formating."
            Href="null"
            contextId="null">
         <nullAction/>
         <list/>
         <nullPerformWhen/>
         <nullList/>
      </item>
      <item
            title="Item with a line break"
            description="<form><p>Line one
<br/>Line two with a special &lt; char in the middle.
</p></form>"
            Href="null"
            contextId="null">
         <nullAction/>
         <list/>
         <nullPerformWhen/>
         <nullList/>
      </item>
      <item
            title="Item with all the special charaters"
            description="<form><p>&lt; - less than
<br/><b>Special characters</b><br/>less than:   &lt;
<br/>greater than: &gt;
<br/>ampersand:   &amp;
<br/>apostrophe:  &apos;
<br/>quote:       &quot;
<br/></p></form>"
            Href="null"
            contextId="null">
         <nullAction/>
         <list/>
         <nullPerformWhen/>
         <nullList/>
      </item>
      <item
            title="Item with one less than"
            description="<form><p><b>&lt;</b></p></form>"
            Href="null"
            contextId="null">
         <nullAction/>
         <list/>
         <nullPerformWhen/>
         <nullList/>
      </item>
      <item
            title="Item with one greater than"
            description=">"
            Href="null"
            contextId="null">
         <nullAction/>
         <list/>
         <nullPerformWhen/>
         <nullList/>
      </item>
      <item
            title="Item with one bold less than"
            description="<form><p><b>&lt;</b></p></form>"
            Href="null"
            contextId="null">
         <nullAction/>
         <list/>
         <nullPerformWhen/>
         <nullList/>
      </item>
      <item
            title="Item with one bold greater than"
            description="<form><p><b>&gt;</b></p></form>"
            Href="null"
            contextId="null">
         <nullAction/>
         <list/>
         <nullPerformWhen/>
         <nullList/>
      </item>
      <item
            title="Item with one less than before a break"
            description="<form><p>&lt;<br/></p></form>"
            Href="null"
            contextId="null">
         <nullAction/>
         <list/>
         <nullPerformWhen/>
         <nullList/>
      </item>
      <item
            title="Item with one less than after a break"
            description="<form><p><br/>&lt;</p></form>"
            Href="null"
            contextId="null">
         <nullAction/>
         <list/>
         <nullPerformWhen/>
         <nullList/>
      </item>
      <item
            title="Item with one less than between breaks"
            description="<form><p><br/>&lt;<br/></p></form>"
            Href="null"
            contextId="null">
         <nullAction/>
         <list/>
         <nullPerformWhen/>
         <nullList/>
      </item>
      <item
            title="Item with one greater than before a break"
            description="<form><p>&gt;<br/></p></form>"
            Href="null"
            contextId="null">
         <nullAction/>
         <list/>
         <nullPerformWhen/>
         <nullList/>
      </item>
      <item
            title="Item with one greater than after a break"
            description="<form><p><br/>&gt;</p></form>"
            Href="null"
            contextId="null">
         <nullAction/>
         <list/>
         <nullPerformWhen/>
         <nullList/>
      </item>
      <item
            title="Item with one greater than between breaks"
            description="<form><p><br/>&gt;<br/></p></form>"
            Href="null"
            contextId="null">
         <nullAction/>
         <list/>
         <nullPerformWhen/>
         <nullList/>
      </item>
   </list>
</cheatsheet>
